### PR TITLE
Rename setLastRenderedPoint to updateInkTrailStartPoint

### PR DESCRIPTION
**Pull request by mabian-ms**
Issue #7 was opened to change the name of setLastRenderedPoint to something that is clearer to what exactly it is, and @dlibby- and I landed on updateInkTrailStartPoint after offline discussion.

### CODE REVIEW

**Comment by mabian-ms** [comment]
@dlibby- PTAL when you can, thanks.

File: README.md [hunk 1]
```diff
@@ -153,15 +153,15 @@ interface DelegatedInkTrailPresenter : InkPresenter {
...
+The `DelegatedInkTrailPresenter` `updateInkTrailStartPoint` method is main addition of this proposal. Authors should use this method to indicate to the User Agent which PointerEvent was used as the last rendered point for the current frame. The PointerEvent passed to `updateInkTrailStartPoint` must be a trusted event and should be the last point that was used to by the application to render ink to the view. `updateInkTrailStartPoint` also accepts all relevant properties of rendering the ink stroke so that the User Agent can closely match the ink rendered by the application. The `diameter` passed as part of `InkTrailStyle` is to indicate the width of the ink trail drawn by the User Agent.
```
Comment (dlibby-) [inline README.md:156]: lets add one more sentence after the first one that mentions something like

"This PointerEvent (and related style information) will be used by the User Agent/Operating system as the starting point for which the delegated trail will be rendered for the subsequent frame that is produced."

Comment (dlibby-) [reply README.md:156]: Or maybe this can be merged with the `updateInkTrailStartPoint` also accepts... sentence somehow

Comment (mabian-ms) [reply README.md]: Added, and moved the one awkward sentence about diameter into its own section about `InkTrailStyle` and how that is extensible for more properties in the future. Let me know what you think.

**Pull request merged**